src/js/game/LevelMVC/FacingDirection.js

```diff
@@ -13,6 +13,14 @@ const FacingDirection = Object.freeze({
     }
   },
 
+  left: function (facing) {
+    return (facing + 3) % 4;
```

Comment:
Nit: let's deduplicate this with `turn`.  Either implement as `return FacingDirection.turn(facing, 'left')` or update `turn` to:

```js
turn: function (facing, rotation) {
  return FacingDirection[rotation](facing);
}
```

Reply:
Do we want opposite to be lumped in there as well? They all feel very similar to me. "Given the direction I'm facing, give me another direction relative to that one.
>left of
>right of
>reverse of

Changing the left/right to call turn is easy but let me know if you feel reversing should be lumped in there as well.